missouri_state_thesis_main.aux

```diff
@@ -0,0 +1,45 @@
+\relax 
```

Comment:
The generated files should not be pushed to this repository. It would be good to add them to the `.gitignore` file.

See here: https://www.gitignore.io/api/latex